packs/st2/actions/kv_get.py

```diff
@@ -13,7 +13,7 @@ def run(self, key, decompress=False):
         _key = self.client.keys.get_by_name(key)
 
         if not _key:
-            return None
+            raise Exception("Key does not exist")
```

Comment:
> There are also some other kv.get* actions which we should also update for consistency (kv_get_object.py).

Ok, I will add.

> Also, this will affect all the existing work-flows which use this action when the key doesn't exist - previously it went to the "on-success" branch, now it will go into "on-error".

Yeah, I know. :/ Not too thrilled about potential regressions, but the behavior is wrong and I'd rather break that while we still have some leeway (pre 1.0)

CLI also has the same behavior (returns `0` when key does not exist). A bit of a tangent, but also discovered.